### PR TITLE
[Layout][Admin] Create Add Category Page

### PR DESCRIPTION
**Pull request by kaz-m-93**
create add category page
asana:
https://app.asana.com/0/1199094899705013/1199181177413494

### CODE REVIEW

File: app/assets/stylesheets/shared.scss [hunk 1]
```diff
@@ -22,6 +22,14 @@
 	display: block;
```
Comment (CharlotteDS) [inline app/assets/stylesheets/shared.scss:22]: Please adjust the indentation in this file.
Use 2 spaces for `.scss` files.

File: app/controllers/admin/categories_controller.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 class Admin::CategoriesController < ApplicationController
   def index
   end
+
+  def new
+  end
```
Comment (CharlotteDS) [inline app/controllers/admin/categories_controller.rb:6]: Please add `@category = Category.new` in this method.

File: app/views/admin/categories/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<% provide(:title, "[admin] new category") %>
+<div class="admin-category-wrapper">
+    <div class="container clearfix">
+        <%= form_for(:category, url: admin_categories_path) do |f| %>
```
Comment (CharlotteDS) [inline app/views/admin/categories/new.html.erb:4]: Forms for namespaced controllers can be adjust to this:
`<%= form_for([:admin, @category]) do |f| %>` 

Reference: https://guides.rubyonrails.org/v5.2/form_helpers.html#dealing-with-namespaces

**Review by CharlotteDS — CHANGES_REQUESTED** [review]

**Comment by kaz-m-93** [comment]
i added category model and fixed param in form_for
asana:
https://app.asana.com/0/1199094899705013/1199181177413494

**Comment by kaz-m-93** [comment]
i fixed the indentaion

**Review by CharlotteDS — APPROVED** [review]

**Pull request merged**